Questionnaire/NHSDigital-Questionnaire-Genomics-Example.json

```diff
@@ -0,0 +1,364 @@
+      "type": "group",
+      "item": [
+        {
+          "linkId": "patientNamecombined",
+          "text": "Patient Name",
+          "type": "string",
+          "required": true
+        },
+        {
+          "linkId": "patientSignature",
+          "text": "Signature",
+          "type": "string",
+          "required": true
+        },
+        {
+          "linkId": "datePatientCompletedForm",
+          "text": "Date",
+          "type": "dateTime",
+          "required": true
+        }
+      ]
+    },
+    {
+      "linkId": "attorney",
+      "text": "If you are signing this form on behalf of someone else (children, adults without capacity or deceased patients) then please sign below.",
+      "type": "group",
+      "item": [
+        {
+          "linkId": "attorneyName",
+          "text": "Parent | Guardian | Consultee name*(please amend as appropriate)",
+          "type": "string",
+          "required": true
+        },
+        {
+          "linkId": "attorneySignature",
+          "text": "Signature",
+          "type": "string",
+          "required": true
+        },
+        {
+          "linkId": "dateAttorneyCompletedForm",
+          "text": "Date",
+          "type": "dateTime",
+          "required": true
+        }
+      ]
+    },
+    {
+      "linkId": "declaration5",
+      "text": "Healthcare professional use only",
+      "type": "group",
+      "item": [
+        {
+          "linkId": "healthcareProfessional",
+          "text": "To be completed by the healthcare professional recording the patient’s choices.",
+          "type": "group",
+          "item": [
+            {
+              "linkId": "patientCategory",
+              "text": "Patient category",
+              "type": "choice",
+              "required": true,
+              "answerValueSet": "https://fhir.nhs.uk/ValueSet/genomics-patience-choice-category"
+            },
+            {
+              "linkId": "testType",
+              "text": "Test type",
+              "type": "choice",
+              "required": true,
+              "answerValueSet": "https://fhir.nhs.uk/ValueSet/genomics-test-types"
+            },
+            {
+              "linkId": "reasonsforChoiceA",
+              "text": "If answer to research choice A is NO",
+              "type": "choice",
+              "required": true,
+              "answerValueSet": "https://fhir.nhs.uk/ValueSet/genomics-patience-choice-researchparticipation"
+            },
+            {
+              "linkId": "remoteConsent",
+              "text": "Remote consent, recorded remotely by clinician, no patient signature",
+              "type": "boolean",
+              "required": true
+            },
+            {
+              "linkId": "responsibleClinician",
+              "text": "Responsible clinician",
+              "type": "string",
+              "required": true
+            },
+            {
+              "linkId": "hospitalNumber",
```

Comment:
This appears to either be the patients hospital number or the practitioners professional code. Can this be clarified?

It may be useful to know if this is a MRN (Patient hospital number), GMC, GMP, etc code (practitioner number)

Reply:
It is the patient MRN. I will change the **link ID** to patientMRN.